### PR TITLE
Milvus : Added properties to dbConfig parameter in function fromExistingCollection

### PR DESCRIPTION
**Pull request by arijit-chowdhury-genea**

### CODE REVIEW

File: langchain/src/vectorstores/milvus.ts [hunk 1]
```diff
@@ -405,6 +405,9 @@ export class Milvus extends VectorStore {
     dbConfig: {
```
Comment (nfcampos) [inline langchain/src/vectorstores/milvus.ts:405]: This should instead be `dbConfig: MilvusLibArgs`

**Review by nfcampos — COMMENTED** [review]
One comment

File: langchain/src/vectorstores/milvus.ts (hunk 1 above)
Comment (arijit-chowdhury-genea) [reply langchain/src/vectorstores/milvus.ts]: I've updated.

**Review by nfcampos — APPROVED** [review]
Thanks!

**Comment by arijit-chowdhury-genea** [comment]
@nfcampos Fixed a formatting issue. Passing all checks.

**Comment by nfcampos** [comment]
@arijit-chowdhury-genea Thanks!

**Pull request merged**